internal/resume/templates/plaintext.go

```diff
@@ -1,47 +1,59 @@
 package templates
 
-import "text/template"
+import (
+  "text/template"
+  "strings"
+	"regexp"
+)
 
 // Plaintext returns the go template of the plaintext resume template
 func Plaintext() *template.Template {
-	tmpl, err := template.New("plaintext").Parse(plaintext)
+  fns := template.FuncMap{"censor": plaintextCensor}
+	tmpl, err := template.New("plaintext").Funcs(fns).Parse(plaintext)
 	if err != nil {
 		panic(err)
 	}
 
 	return tmpl
 }
 
+func plaintextCensor(s string) string {
+  re := regexp.MustCompile(`\|\|.*?\|\|`)
+  //func test(_s string) { return strings.Repeat("#",len(_s)-4) })
+	s = re.ReplaceAllStringFunc(s, func(_s string) string { return strings.Repeat("#",len(_s)-4) })
```

Comment:
In Go, underscores are generally used to signal discarded variables/parameters. Perhaps renaming `s` to `text` and `_s` to `s` would be cleaner? Something similar would need to be done in the latex template's censor function, for consistency.